src/main/java/com/exasol/projectkeeper/XPathSplitter.java

```diff
@@ -0,0 +1,45 @@
+package com.exasol.projectkeeper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class splits xPath expressions at the root level slashes.
+ */
+public class XPathSplitter {
+    private XPathSplitter() {
+        // empty on purpose
+    }
+
+    /**
+     * Split the xPath expression at the root level slashes.
+     * 
+     * @param xPath xPath expression
+     * @return fragments
+     */
+    public static List<String> split(final String xPath) {
```

Comment:
Extract methods please

Reply:
Not relay possible here, since its one algorithem.